frontend/packages/ceph-storage-plugin/integration-tests/views/noobaa-sso.view.ts

```diff
@@ -0,0 +1,9 @@
+import { $, element, by, $$ } from 'protractor';
+
+export const objectServiceLink = element(by.cssContainingText('a', 'Object Service'));
```

Comment:
Expand tests were being written wrt 4.3 and this and other tests wrt 4.2. Why there is inconsistency ?
How this is being planned ?
With latest masterI see few of  these tests will fail, for example here, since the tab name is changed to `OCS Object Service`.

Reply:
For now we are writing tests for 4.3 and 4.2 which support OCS deployment.
The test are not going to get into e2e. We are invoking the test from our ocs-ci for now.